input/fsh/CGExamples.fsh

```diff
@@ -1133,7 +1133,7 @@ Description: "Example of a Therapeutic Implication for Clinical Trial"
 * component[1].code.coding = LNC#81259-4 // "Phenotypic treatment context"
 //#HP:0003002
 * component[1].valueCodeableConcept.coding = 	SCT#254837009 "Breast Cancer"
-* component[2].code.coding = TbdCodesCS#associated-therapy "Associated Therapy"
+* component[2].code.coding = TbdCodesCS#associated-therapy "Genomically linked therapy"
```

Comment:
@khalifa-aly - Why did you make this change? I think we moved away from the 'linked therapy' wording to the 'Associated Therapy' wording?

Reply:
I couldn't remember I made this change. So, we may keep it as Associated Therapy.

Reply:
I'll change it